core-service/src/test/java/com/crmfoodestablishment/coreservice/service/MenuServiceTest.java

```diff
@@ -0,0 +1,147 @@
+package com.crmfoodestablishment.coreservice.service;
+
+import com.crmfoodestablishment.coreservice.dto.MenuDto;
+import com.crmfoodestablishment.coreservice.entity.Menu;
+import com.crmfoodestablishment.coreservice.entity.Season;
+import com.crmfoodestablishment.coreservice.mapper.MenuMapper;
+import com.crmfoodestablishment.coreservice.repository.MenuRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class MenuServiceTest {
+    @InjectMocks
+    private MenuService service;
+
+    @Mock
+    MenuRepository menuRepository;
+
+    @Mock
+    MenuMapper menuMapper;
+
+    @BeforeEach
+    void setUp() {
+        service = new MenuService(menuRepository);
```

Comment:
Could u pls check if we really need this line?

Reply:
I deleted this part of code.